components/Modal/Modal.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./Modal.css" />
+  </head>
+  <body>
+    <h2 class="sm-gutter">Modal</h2>
+    <a class="btn toggle-btn primary-btn sm-gutter md-brr para-sm-xs"
+      >Open Modal
+    </a>
+
+    <div class="backdrop"></div>
+    <div class="modal sm-padding">
+      <div class="content">
+        <h2>Modal Title</h2>
+        <p class="xs-padding para-sm-xs">
+          Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. It has survived not only
+          five centuries, but also the leap into electronic typesetting,
+          remaining essentially unchanged. It was popularised in the 1960s with
+          the release of Letraset sheets containing Lorem Ipsum passages, and
+          more recently with desktop publishing software like Aldus PageMaker
+          including versions of Lorem Ipsum.
```

Comment:
You can add some real content so it will look good to user

Reply:
ok sure